tsdb/engine/tsm1/iterator.gen.go.tmpl

```diff
@@ -108,7 +108,7 @@ type {{.name}}Iterator struct {
 	aux   []cursorAt
 	conds struct {
 		names []string
-		curs  []*bufCursor
+		curs  []cursorAt
```

Comment:
Hm, we might be able to rewrite the condition instead since, in the case of tags, the value is static. Do you want me to see how easy that would be first?